filesource-common/pom.xml

```diff
@@ -52,6 +60,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Why do we need this change?

Reply:
This is to set the maven version to 1.8+

Reply:
Done